runtime/src/tests/availability.rs

```diff
@@ -340,3 +340,22 @@ fn pallet_aggregate() {
         // just checking code builds, hence the pallet is available to the runtime
     });
```

Comment:
missing test for availability of the new pallet you introduced + tests ensuring it is using the correct weights